scripts/benchmark/benchmark_requests.py

```diff
@@ -123,14 +128,30 @@
             else:
                 success.append(0)
 
-    outfile = (
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+    if output:
+        outfile = Path(output)
+    else:
+        filename = (
+            f"benchmark_[t={timestamp}]"
+            f"_[n={num_requests}]"
+            f"_[w={max_workers}]"
+            f"_[r={route}]"
+            f"_[execution_mode={'par' if parallel else 'seq'}]"
+            ".csv"
+        )
+        outfile = RESULTS_DIR / filename
```

Comment:
You can simply compute and use the results dir here, since it is not being used anywhere else. Also, no need to make it a constant.